pkgs/development/libraries/yas/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```

Comment:
Given that it does not call GCC, use `stdenvNoCC`.

Reply:
OK, I have used `stdenvNoCC` now.